dmcontent/metadata.py

```diff
@@ -0,0 +1,40 @@
+class ContentMetadata(object):
```

Comment:
As all the apps are now Python3, do we need to inherit from `object`?

Reply:
I was getting a weird flake8/pep8 thing when I dropped that... So I left it in. 🙈 It wanted me to define an `__init__` method, but it would be empty. I guess I can do it anyway.

Reply:
Ok not getting the problem now. So yes, I'll remove the inherit. 👍